pages/list_verification.py

```diff
@@ -0,0 +1,26 @@
+import os
+from pypom import Page
+import pytest
+import urllib2
+import requests
+
+
+TEST_ENV = os.environ['TEST_ENV']
+
+
+class ListVerificationPage(Page):
+
+    URL_TEMPLATE = 'https://shavar.stage.mozaws.net/list?client=foo&appver=1&pver=2.2'
```

Comment:
i recall you mentioned (and demonstrated) that URL_TEMPLATE is somehow required by the page object thang.   However since, in truth, we're pulling this value from our prefs.ini file when we need it, it seems a bit misleading to just leave this here as-is.  we can merge it and fix it later, but we should perhaps figure out how to either dispense with this var altogether or how to use it properly since maybe we're not??